jac-uk/admin
# #1775 Additional Personal Details Fields

## What's included?
Add the following fields to the personal details and custom report:
- Middle name(s) (optional)
- Suffix (optional)
- Previous known name(s) (optional)
- Professional name (optional)

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
1. Go to the personal details of an application and check the above fields are saved properly.
2. Go to the custom report and check if these above fields can be selected.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Screenshots:

<img width="949" alt="admin #1775 Additional Personal Details Fields_1" src="https://user-images.githubusercontent.com/79906532/200361627-0a948e7d-ddd1-4c54-84f7-2d891ee9d26e.png">

<img width="1241" alt="admin #1775 Additional Personal Details Fields_2" src="https://user-images.githubusercontent.com/79906532/200361654-18176b34-a2af-4cb3-bf96-f2b58373044e.png">

## Related permissions
- No permission changes required

---
PREVIEW:DEVELOP
